docs/Analyzer Configuration.md

```diff
@@ -408,4 +408,47 @@ Option Values:
 
 Default Value: `Heuristic`.
 
-Example: `dotnet_code_quality.CA1712.enum_values_prefix_trigger = AnyEnumValue`
\ No newline at end of file
+Example: `dotnet_code_quality.CA1712.enum_values_prefix_trigger = AnyEnumValue`
+
+### Exclude indirect base types
+Option Name: `exclude_indirect_base_types`
+
+Configurable Rules: [CA1710](https://docs.microsoft.com/visualstudio/code-quality/ca1710-identifiers-should-have-correct-suffix)
+
+Option Values: `true` or `false`
+
+Default Value: `false`
+
+Example: `dotnet_code_quality.CA1710.exclude_indirect_base_types = true`
+
+For example, consider the code below:
+```csharp
+// An issue is always raised on this type because the suffix should be 'Exception'.
+public class MyBaseClass : Exception, IEnumerable
+{
+   // code omitted for simplicity
+}
+
+// If the option is enabled no issue is raised on 'MyClass'; otherwise an issue will
+// suggest to add the 'Exception' suffix.
+public class MyClass : MyBaseClass
+{
+   // code omitted for simplicity
+}
+```
+
+### Additional required suffixes
+Option Name: `additional_required_suffixes`
+
+Configurable Rules: [CA1710](https://docs.microsoft.com/visualstudio/code-quality/ca1710-identifiers-should-have-correct-suffix)
+
+Option Values: List (separated by '|') of fully qualified type name followed by colon and the required suffix.
```

Comment:
I would prefer we keep re-using the existing supported formats for type name specification using documentation comment ID. See https://github.com/dotnet/roslyn-analyzers/blob/master/docs/Analyzer%20Configuration.md#excluded-type-names-with-derived-types.

Reply:
I am not sure to get you... Are you suggesting to use this syntax for the type name but keep the `:SUFFIX` part?

Reply:
Yes, sorry I had forgotten that you need to have a suffix for each type name. Please use the same format for type name specification, though you would need to write your own custom parse method for each type name + suffix pair.

Reply:
Do we need to add documentation about the type name format before the separator character? Similar to https://github.com/dotnet/roslyn-analyzers/blob/master/docs/Analyzer%20Configuration.md#excluded-type-names-with-derived-types:

```
Allowed type name formats:

- Type name only (includes all types with the name, regardless of the containing type or namespace)
- Fully qualified names in the symbol's documentation ID format: https://github.com/dotnet/csharplang/blob/master/spec/documentation-comments.md#id-string-format with an optional "T:" prefix.
```